NAMESPACE

```diff
@@ -2,5 +2,4 @@ export(Correlation)
 export(RegressionLinear)
 export(RegressionLogistic)
 export(CorrelationBayesian)
-export(CorrelationBayesianPairs)
```

Comment:
Did I forget something here? In principle, I removed CorrelationBayesianPairs already, but I indeed did not consider any namespaces.

Reply:
I think that was just deleted automatically when I installed the module (with regernerate meta data every time). I can put it back, but I thought we do not need it anymore, right?

Reply:
We don't need it, I was just curious. So what you've done is correct.